nx.js

```diff
@@ -0,0 +1,10 @@
+require('./patch');
+
+/** @type {import('eslint').Linter.Config} */
+module.exports = {
+	extends: [
+		'plugin:@next/next/core-web-vitals',
+		'plugin:@nrwl/nx/typescript',
+		'./next.js',
+	],
+};
```

Comment:
Lets try and not pollute our ruleset with everything from nx - but instead just pick out the [valuable rules](https://github.com/nrwl/nx/blob/master/packages/eslint-plugin-nx/src/index.ts):

```suggestion
const workspaceRules = require('@nrwl/eslint-plugin-nx/src/resolve-workspace-rules');
const { RULE_NAME, default: enforceModuleBoundaries } = require('@nrwl/eslint-plugin-nx/src/rules/enforce-module-boundaries');

require('./patch');

/** @type {import('eslint').Linter.Config} */
module.exports = {
	extends: [
		'plugin:@next/next/core-web-vitals',
		'./next.js',
	],
	rules: {
		[RULE_NAME]: enforceModuleBoundaries,
		...workspaceRules,
	},
};
```